src/components/extras/NavbarBody.vue

```diff
@@ -32,26 +31,107 @@
                 <b-navbar-item tag="router-link" :to="{ name: 'Faq' }">
                     Faq
                 </b-navbar-item>
+
+                <template v-if="isAuthReady">
```

Comment:
why is this a part of this PR.

Reply:
when user comes 2nd time and not logged out before . we are checking in firebase whether it was logged out in previous session ...if yes then show login button or show the user image .